project/Build.scala

```diff
@@ -269,6 +270,16 @@ object ScaldingBuild extends Build {
       "org.apache.hadoop" % "hadoop-core" % hadoopVersion % "provided",
       "org.slf4j" % "slf4j-api" % slf4jVersion,
       "org.slf4j" % "slf4j-log4j12" % slf4jVersion % "provided"
+    ) ++ (
+      if(isScala210x(scalaVersion.value)) {
+        Seq(
+          "com.chuusai" % "shapeless_2.10.4" % shapelessVersion
```

Comment:
My understanding (based on the [README](https://github.com/milessabin/shapeless)) is that shapeless 2.1.0 is only compatible with 2.11.x and 2.10.4, but I might be mistaken.  I think the relevant bits are as follows:

>Builds are available for Scala 2.11.x and for Scala 2.10.4
>...
>Note that for Scala 2.10.4 you must provide an explicit Scala version suffix to your shapeless dependency, and it is recommanded to add the macro paradise plugin to your build, for some macros provided by shapeless to work smoothly

For shapeless 2.0.0, they have the following comments:

>Builds are available for Scala 2.11.x and for Scala 2.10.4.
>...
>For Scala 2.10.x you must specify a Scala version of at least 2.10.2, and add either `cross CrossVersion.full` or provide an explicit Scala version suffix to your shapeless dependency
>...
>Note that Scala 2.10.x releases are compatible with each other starting from 2.10.2, so a mismatch in minor versions above would be fine.